telegram/ext/conversationhandler.py

```diff
@@ -354,12 +354,12 @@ def update_state(self, new_state, key):
             if self.persistent:
                 self.persistence.update_conversation(self.name, key, new_state)
 
-    def _trigger_timeout(self, bot, job):
+    def _trigger_timeout(self, dispatcher):
```

Comment:
You're pretty close. `bot, job` does indeed have to go away. But instead of `dispatcher` it should take `context`